crates/db_schema/src/impls/post.rs

```diff
@@ -379,39 +374,34 @@ impl PostRead {
 impl PostHide {
   pub async fn hide(
     pool: &mut DbPool<'_>,
-    post_ids: HashSet<PostId>,
+    post_id: PostId,
     person_id: PersonId,
   ) -> Result<usize, Error> {
     let conn = &mut get_conn(pool).await?;
 
-    let forms = post_ids
-      .into_iter()
-      .map(|post_id| {
-        (
-          PostHideForm { post_id, person_id },
-          post_actions::hidden.eq(now().nullable()),
-        )
-      })
-      .collect::<Vec<_>>();
+    let form = (
+      &PostHideForm { post_id, person_id },
+      post_actions::hidden.eq(now().nullable()),
+    );
     insert_into(post_actions::table)
-      .values(forms)
+      .values(form)
       .on_conflict((post_actions::person_id, post_actions::post_id))
       .do_update()
-      .set(post_actions::hidden.eq(now().nullable()))
+      .set(form)
```

Comment:
Is this change right? It looks like it was taking a `bool` before but a `(PostHideForm, bool)` now. Should it be `form.1` since that's using the same expression?